Gemfile

```diff
@@ -18,6 +18,8 @@ gem "coffee-rails"
 gem "sass-rails"
 gem "turbolinks", "~> 5"
 
```

Comment:
Do we need a blank line here then? Does it go with the group above?

Reply:
Yes, I think this can be in the same group